crypto/src/mls/conversation/decrypt.rs

```diff
@@ -326,10 +331,26 @@ pub mod tests {
                             .await
                             .unwrap();
                         // Now Debbie should be in members and not Charlie
-                        assert!(alice_central[&id].members().get(&debbie.id).is_some());
-                        assert!(alice_central[&id].members().get(&charlie.id).is_none());
+                        assert!(alice_central
```

Comment:
It could probably be worth adding a test helper to dodge this boilerplate

Reply:
I thought about it too - our previous helper in the form of implementing `Index`/`IndexMut` couldn't do it as this is `async`.

Reply:
Yes obviously not relying on `Index(Mut)` ; more something like `MlsGroup::find_member(id: &[u8])`

Reply:
I added the following API to `MlsCentral`'s test_utils and called it in all tests: 

It's panicky but whatever

```rust
pub async fn get_conversation_unchecked(
    &mut self,
    conv_id: &ConversationId,
) -> async_lock::RwLockWriteGuard<'_, MlsConversation>;
```